fs/dafsserver/dafsserver.cpp

```diff
@@ -4362,9 +4363,13 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                     }
                     catch (IException *e)
                     {
-                        EXCLOG(e,"CRemoteFileServer");
-                        e->Release();
-                        continue;
+                        exception.setown(e);
+                    }
+                    if (exception)
```

Comment:
You could include this code within the try and then avoid assigning to exception.  I don't mind either way.

Reply:
I'll leave this one as is, partly because in another PR, I want to call something here, and don't want to call in catch